<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObdScan</class>
 <widget class="QMainWindow" name="ObdScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelVoltTitle">
          <property name="text">
           <string>Volt: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelVolt">
          <property name="text">
           <string>0.0 V</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelRpmTitle">
          <property name="text">
           <string>Rpm:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelRpm">
          <property name="text">
           <string>0 rpm</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelLoadTitle">
          <property name="text">
           <string>Engine Load:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelLoad">
          <property name="text">
           <string>0 %</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelSpeedTitle">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelSpeed">
          <property name="text">
           <string>0 km/h</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelOilTempTitle">
          <property name="text">
           <string>Oil Temp:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelOilTemp">
          <property name="text">
           <string>0 C째</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelIntakeAirTempTitle">
          <property name="text">
           <string>Intake Air Temp:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="labelIntakeAirTemp">
          <property name="text">
           <string>0 C째</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelCoolantTitle">
          <property name="text">
           <string>Coolant Temp:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelCoolant">
          <property name="text">
           <string>0 C째</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labelMafAirFlowTitle">
          <property name="text">
           <string>Maf Air Flow:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="labelMafAirFlow">
          <property name="text">
           <string>0 g/s</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="labelManifoldPressureTitle">
          <property name="text">
           <string>Manifold Pressure:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="labelManifoldPressure">
          <property name="text">
           <string>0 kPa</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="labelFuelRailLowPressureTitle">
          <property name="text">
           <string>Fuel Rail Low:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="labelFuelRailLowPressure">
          <property name="text">
           <string>0 kPa </string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="labelFuelRailHighPressureTitle">
          <property name="text">
           <string>Fuel Rail High:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="labelFuelRailHighPressure">
          <property name="text">
           <string>0 kPa</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="labelFuelPressureTitle">
          <property name="text">
           <string>Fuel Pressure:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="labelFuelPressure">
          <property name="text">
           <string>0 kPa</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelAmbientAirTempTitle">
          <property name="text">
           <string>Ambient Air Temp:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="labelAmbientAirTemp">
          <property name="text">
           <string>0 C째</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
